Dockerfile.rhel7

```diff
@@ -18,7 +18,11 @@ RUN yum install -y yum-utils --setopt=tsflags=nodocs && \
     yum-config-manager --enable rhel-server-rhscl-7-rpms && \
     yum-config-manager --enable epel && \
     yum install -y --setopt=tsflags=nodocs python-pip docker && \
-    yum clean all
+    yum clean all && \
+    curl -L https://github.com/openshift/origin/releases/download/v1.0.2/openshift-origin-v1.0.2-325c7b7-linux-amd64.tar.gz | tar -zx && \
```

Comment:
Right, installing from upstream source is not ideal. An oc RPM pkg is coming soon, I believe.